src/Core/src/Platform/iOS/MauiUIApplicationDelegate.Menu.cs

```diff
@@ -60,7 +61,9 @@ public override void BuildMenu(IUIMenuBuilder builder)
 						}
 					}
 					else
+					{
 						window = activeWindowScenes[0].KeyWindow;
```

Comment:
I think here we'll need to add an if statement for "15" 

KeyWindow is only available in 15
https://developer.apple.com/documentation/uikit/uiwindowscene/keywindow

So, if it's lower than 15 we'll want to probably check the windows array and return that
https://developer.apple.com/documentation/uikit/uiwindowscene/windows

Reply:
@PureWeen, I have made the changes as per your suggestion. Could you please review it now?